lib/barbeque/worker.rb

```diff
@@ -33,34 +31,35 @@ def self.run(
     end
 
     def initialize
-      @queue_name = ENV['BARBEQUE_QUEUE'] || DEFAULT_QUEUE
+      @command =
+        case worker_id
```

Comment:
It's okay to fix this worker process role mapping at the first time, but it might not be the best configuration to fully utilize resources. Do you have any idea about monitoring capacity utilization for each worker? It would be good to have such metrics in dashboard in the future.